### DISCUSSION TITLE
RP2350 - PSRAM

### DISCUSSION THREAD

**Discussion by elral** [earlephilhower/arduino-pico#2439] [General]
I tried to use PSRAM according the documentation, but compilation with Arduino IDE and platformIO failed:
`'pmalloc' was not declared in this scope; did you mean 'pvalloc'?`
```int *buff;
   buff = (int *)pmalloc(10000 * sizeof(*buff));
```

Using `rp2040.getPSRAMSize()`  results in `undefined reference to '__psram_size'  collect2.exe: error: ld returned 1 exit status`
also with both IDE's.
`Serial.print(rp2040.getPSRAMSize());`

Might it that I do not have the latest version?
I have defined in Arduino IDE
`https://github.com/earlephilhower/arduino-pico/releases/download/global/package_rp2040_index.json`

My paltformIO.ini is:
```platform = https://github.com/maxgerhardt/platform-raspberrypi.git
board = rpipico2
framework = arduino
board_build.core = earlephilhower```

**Comment by PontusO** [comment]
The rpipico2 does not come with any PSRAM and therefore no support for that is included.

**Reply by elral** [reply]
Uuups, what a Fauxpax. Sorry for this. Playing around with too much boards for now...
